warp10/src/main/java/io/warp10/script/ext/stackps/WSINFO.java

```diff
@@ -0,0 +1,96 @@
+//
+//   Copyright 2020  SenX S.A.S.
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+
+package io.warp10.script.ext.stackps;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import io.warp10.script.NamedWarpScriptFunction;
+import io.warp10.script.WarpScriptException;
+import io.warp10.script.WarpScriptStack;
+import io.warp10.script.WarpScriptStackFunction;
+import io.warp10.script.WarpScriptStackRegistry;
+
+public class WSINFO extends NamedWarpScriptFunction implements WarpScriptStackFunction {
+  
+  public static final String[] EXPOSED_ATTRIBUTES = new String[] {
+      WarpScriptStack.ATTRIBUTE_CREATION_TIME,
+      WarpScriptStack.ATTRIBUTE_FETCH_COUNT,
+      WarpScriptStack.ATTRIBUTE_GTS_COUNT,
+      WarpScriptStack.ATTRIBUTE_MACRO_NAME,
+      WarpScriptStack.ATTRIBUTE_NAME,
+      WarpScriptStack.ATTRIBUTE_SECTION_NAME,
```

Comment:
Missing: 
     StackPSWarpScriptExtension.ATTRIBUTE_SESSION

(could be relocated in WarpScriptStack maybe ?)

Reply:
nope since it is added by an extension it is more logical to have it in the extension itself

Reply:
Yes for location, but having session name among infos in WSPS is great too.